packages/SwingSet/src/initializeSwingset.js

```diff
@@ -280,6 +280,7 @@ export async function initializeSwingset(
     creationOptions: {
       enablePipelining: true,
       enableSetup: true,
+      managerType: 'local',
```

Comment:
let's add a note to remind ourselves that the comms vat is made local because it uses `setup` instead of `buildRootObject`, and we don't plan to support that in the non-local workers any time soon